gravity.sh

```diff
@@ -510,9 +510,11 @@ gravity_ParseBlacklistDomains() {
   
   if [[ -f "${piholeDir}/${whitelistMatter}" ]]; then
     gravity_ParseDomainsIntoHosts "${piholeDir}/${whitelistMatter}" "${piholeDir}/${accretionDisc}"
+    grep -c "^" "${piholeDir}/${whitelistMatter}" > "${piholeDir}/numBlocked" 2> /dev/null
   else
     # There was no whitelist file, so use preEventHorizon instead of whitelistMatter.
     gravity_ParseDomainsIntoHosts "${piholeDir}/${preEventHorizon}" "${piholeDir}/${accretionDisc}"
+    grep -c "^" "${piholeDir}/${preEventHorizon}" > "${piholeDir}/numBlocked" 2> /dev/null
```

Comment:
Are these changes supposed to be in this PR?

Reply:
Huh. Good catch. Possibly went back by one too many when i squashed the commits. 
Identical to : https://github.com/pi-hole/pi-hole/pull/2040/commits/ff47ab12fbc8bc9164c4c75d92f4cfba94d202d2